### PR TITLE
feat: #1057 Align design for Accordion component

### PR DESCRIPTION
**Pull request by algrinkevich**
Aligned the design for Accordion component.

 P.S. I changed "iconSizeMedium" from 25px to 24px global size, as 25px of size doesn't exist in the design, but 24px does (as the designer said).

### CODE REVIEW

File: mw-webapp/src/globalStore/ThemeStore.ts [hunk 1]
```diff
@@ -40,6 +40,10 @@ const themedVariables: Record<string, Record<Theme, string>> = {
     [Theme.DARK]: "#686085",
     [Theme.LIGHT]: "#F2EFF9",
   },
+  hoverBgColor: {
+    [Theme.DARK]: "#A8A1DD",
+    [Theme.LIGHT]: "#A8A1DD",
+  },
```
Comment (tritonJS826) [inline mw-webapp/src/globalStore/ThemeStore.ts:46]: Do we have such variable in tokens?

Comment (algrinkevich) [reply mw-webapp/src/globalStore/ThemeStore.ts:46]: Yes

File: mw-webapp/src/component/accordion/AccordionContent/AccordionContent.module.scss [hunk 2]
```diff
@@ -1,5 +1,7 @@
 @import "src/styles/variables";
 
+$textLineHeight: 1.3;
+
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/accordion/AccordionContent/AccordionContent.module.scss:4]: let's use our  global token with lineheight 1.4

File: mw-webapp/src/component/accordion/AccordionContent/AccordionContent.module.scss [hunk 3]
```diff
@@ -1,5 +1,7 @@
 @import "src/styles/variables";
 
+$textLineHeight: 1.3;
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/accordion/AccordionContent/AccordionContent.module.scss:3]:
```suggestion
```

File: mw-webapp/src/component/accordion/AccordionContent/AccordionContent.module.scss [hunk 4]
```diff
@@ -13,7 +15,8 @@
 }
 
 .accordionContentText {
-  padding: 15px 20px;
+  padding: $paddingSmall 0 $paddingMedium 0;
+  line-height: $textLineHeight;
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/accordion/AccordionContent/AccordionContent.module.scss:19]:
```suggestion
  line-height: $lineHeightPrimary;
```

**Comment by tritonJS826** [comment]
Thank a lot for the images in your PR, it is much easier to see visual changes with them. 
Discuss with @vikpak89 accordion view, I thin you both can make it a little bit more beautiful :)

**Pull request merged**